resources/assets/less/app/kanban.less

```diff
@@ -54,6 +54,110 @@
   }
 }
 
+
+/* navpill */
+.kanban-nav {
+	position: relative;
+	margin: 0px 0;
+}
+.kanban-pill-tag {
+	color: #afafaf;
+	padding-left: 10px !important;
+	padding-right: 10px !important;
+}
+.kanban-nav ul {
+	margin: 0;
+	padding: 0;
+}
+.kanban-nav li {
+	margin: 0 5px 10px 0;
+	padding: 0;
+	list-style: none;
+	display: inline-block;
+}
+.kanban-nav a {
```

Comment:
Regarding less features:
I had no experience with less until now. So it's basically a completely new path for me.

I'll try to learn about its syntax while using it. So it might not be perfect from the start.
But I'll try :)